alpine/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM puppet/puppet-agent-alpine
+
+RUN apk add --update git \
+    && git clone https://github.com/jpogran/puppet-vscode.git
```

Comment:
URL needs to be updated before merge

Reply:
Updated the url and the entrypoint as needed.